BuildDetails.md

```diff
@@ -0,0 +1,39 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how the project is built and why it is built that way.
+
+### Table of Contents
```

Comment:
If something is a documented entity already, like minification or PhantomJS, it's sufficient to provide links to their readmes instead of re-explaining the wheel.